docs/usage.md

```diff
@@ -307,24 +96,6 @@ Specify a .tsv file containing the MHC class 1 alleles of your probes as well as
 | MM15_Melanom | A*03:01;A*68:01;B*27:05;B*35:03;C*02:02;C*04:01 |HLA-DRB1*01:01;HLA-DQB1*03:19;HLA-DQA1*05:01|
 | MM17_Melanom | A*02:01;B*07:01;B*26:01;C*11:01;C*01:01         |HLA-DRB1*01:02;HLA-DRB3*02:02;HLA-DRB4*01:03|
 
```

Comment:
This should also be removed. If you want to keep some extra docs in this document, maybe you can change the title of the section to something like `Input files information`?